trajectory: [[1.,1.,2.,0.],
    [1., 4.5, 1., 0.], 
    [1., 4.5, 1., 90.], 
    [1., 4.5, 1., 180.], 
    [1., 4.5, 1., 270.], 
    [1., 4.5, 1., 0.], 
    [1., 9.5, 1., 0.], 
    [4., 9.5, 1.5, 0.],  
    [4., 9.5, 1.5, 90.],  
    [4., 9.5, 1.5, 180.],  
    [4., 9.5, 1.5, 270.],  
    [4., 9.5, 1.5, 0.],  
    [3., 11., 0.4, 0.],  
    [3., 11., 0.4, 90.], 
    [3., 11., 0.4, 180.], 
    [3., 11., 0.4, 270.], 
    [3., 11., 0.4, 0.], 
    [6., 9.5, 0.5, 0.],
    [6., 6.5, 0.7, 0.],  
    [6., 6.5, 0.7, 90.],  
    [6., 6.5, 0.7, 180.],  
    [6., 6.5, 0.7, 270.],  
    [6., 6.5, 0.7, 0.],  
    [6., 6.5, 1., 0.],  
    [6.5, 5.5, 1.5, 0.], 
    [6.5, 5.5, 1.5, 90.], 
    [6.5, 5.5, 1.5, 180.], 
    [6.5, 5.5, 1.5, 270.], 
    [6.5, 5.5, 1.5, 0.], 
    [5.5, 6.5, 1.5, 0.],  
    [7., 11., 1.5, 0.]]
camera_pose: [-0.158979,-0.04405,0.045074,0.,0.,0.]
focal_length: 277.191356
target_radius: 0.2
